centos7/.bazelrc

```diff
@@ -0,0 +1,17 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
```

Comment:
License here should be the google wide one.

Reply:
fixed